draft-ietf-teas-actn-poi-applicability.txt

```diff
@@ -1549,35 +1558,32 @@ Internet-Draft                 ACTN POI                       July 2023
    o  Inter-domain Ethernet links suppoting inter-domain IP links
       between two adjancent IP domains;
 
-   o  Cross-layer links between an an IP domain and an adjacent optical
-      domain;
-
-   o  Access links between a CE device and a PE router.
-
-   All the three types of links are Ethernet links.
 
 
 
 
+Peruzzini et al.        Expires July 30, 2024                 [Page 29]
+
+Internet-Draft                 ACTN POI                    January 2024
 
 
+   o  Cross-layer links between an an IP domain and an adjacent optical
+      domain;
 
-Peruzzini et al.       Expires January 7, 2024                [Page 29]
-
-Internet-Draft                 ACTN POI                       July 2023
+   o  Access links between a CE device and a PE router.
 
+   All the three types of links are Ethernet links.
 
    There are two possible models to report the access links between CEs
-   and PEs: the Ethernet Topology Model, defined in [CLIENT-TOPO], or
-   the Service Attachment Points (SAP) Model, defined in  [RFC9408].
-   Clarifying the relationship between these two models has been
-   identified as a gap.
+   and PEs: the TE Topology Model, defined in [RFC8795], or the Service
+   Attachment Points (SAP) Model, defined in  [RFC9408]. Clarifying the
+   relationship between these two models has been identified as a gap.
 
    It is worth noting that the P-PNC may not be aware whether an
    Ethernet interface terminates a cross-layer link, an inter-domain
-   Ethernet link or an access link. The Ethernet Topology Model supports
-   the discovery for all these types of links with no need for the P-PNC
-   to know the type of inter-domain link.
+   Ethernet link or an access link. The TE Topology Model supports the
+   discovery for all these types of links with no need for the P-PNC to
+   know the type of inter-domain link.
```

Comment:
did we define in some part cross-layer link, inter-domain Ethernet link and access links (CE-PE Ethernet links)? Again I am not sure that RFC8795 (TE-TOPO) is of any help here for cross-layer links and for access links as we need to understand that these hard UNI ports so in principle we would need SAP + attachment circuit network data model mentioned above, correct?

Reply:
Agreed that TE topology can be used for inter-domain and cross-layer link discovery

For access links, see: https://github.com/IETF-TEAS-WG/actn-poi/pull/115/files?w=0#r1487671518